docs/Using Price Feeds/feed-registry-functions.md

```diff
@@ -0,0 +1,386 @@
+---
+layout: nodes.liquid
+section: ethereum
+date: Last Modified
+title: "Feed Registry Functions"
```

Comment:
For consistency , we should:

- name it "Feed Registry API Reference"
- file name: "feed-registry-api-reference.md"
- link : /docs/feed-registry-api-reference/